css/design_system/components/nav.scss

```diff
@@ -1,55 +1,54 @@
-:root {
-  --nav-height: 4rem;
-}
-
 /**
  * @title Primary Nav
  * @category components
- * @status early draft
- * @description Navigation component w/ primary and secondary variants.
+ * @component_type nav
+ * @description This navigation component can be used with or without the menu.
  * @example
- * <nav class="nav nav-primary">
- *   <ul>
- *     <li>Nav element 1</li>
- *     <li>Nav element 2</li>
+ * <nav class="s-nav">
+ *   <a class="s-nav-icon" href="/" aria-label="Home">
+ *     <img class="s-nav-icon" src="https://assets.turing.edu/turing-school-mark-gray.png" alt="" />
```

Comment:
I think we could use `{{site.assets_url}}` here instead of `https://assets.turing.edu`.

Reply:
Also see same note as above re: a11y for image links

Reply:
Ok will all the sites have access to the site.assets_url ? I guess I took this out initially to use the url from assets page, so it's a bit more generic (so when group projects like Mesh use Savile, this is just there, and site.assets_url doesn't confuse them. I know they aren't the main users, though). Anyways, that's where my brain was.

Reply:
Oh well lucky enough they won't ever actually see `{{site.assets_url}}` because by the time Jekyll renders the docs page it will already have perform the interpolation, so anyone visiting our site will always see the full URL.